CMakeLists.txt

```diff
@@ -785,6 +806,48 @@ if (NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL)
   endif ()
 endif ()
 
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/harfbuzz/harfbuzzConfigVersion.cmake"
+  VERSION ${HB_VERSION}
+  COMPATIBILITY AnyNewerVersion
+)
+
+install(TARGETS harfbuzz EXPORT harfbuzzTargets
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION bin
+  INCLUDES DESTINATION include/harfbuzz
```

Comment:
Usually it's just `include`

Reply:
`pkg-config harfbuzz --cflags` returns include/harfbuzz so I was mirroring that.